scripts/quests/sandoria/Her_Majesty_S_Garden.lua

```diff
@@ -0,0 +1,94 @@
+-----------------------------------
+-- Her Majestys Garden
+-----------------------------------
+-- Log ID: 0, Quest ID: 62
+-- Chalvatot : !pos -105 0.1 72 233
+-- Reward : Map of the Northlands
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.SANDORIA, xi.quest.id.sandoria.HER_MAJESTY_S_GARDEN)
+
+quest.reward =
+{
+    fame     = 30,
+    fameArea = xi.fameArea.SANDORIA,
+    keyItem  = xi.ki.MAP_OF_THE_NORTHLANDS_AREA,
+    gil      = 2000,
+    xp       = 2000,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:getFameLevel(xi.fameArea.SANDORIA) >= 4
+        end,
+
+        [xi.zone.CHATEAU_DORAGUILLE] =
+        {
+            ['Chalvatot'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 0 and player:getNation() ~= xi.nation.SANDORIA then -- Non sandy folk gets a CS prior to getting acceptance
```

Comment:
These conditions should be organized a bit differently for readability.  You have two 0 checks in the same level

Reply:
Fair enough. Swapped around. Thanks.